src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown

```diff
@@ -249,48 +274,82 @@ navigator.bluetooth.requestDevice({ filters: [{ services: ['heart_rate'] }] })
 .then(service => service.getCharacteristic('heart_rate_measurement'))
 .then(characteristic => {
   return characteristic.startNotifications()
-  .then(() => {
+  .then(_ => {
     characteristic.addEventListener('characteristicvaluechanged',
-      handleNotifications);
+                                    handleCharacteristicValueChanged);
   });
 })
-.then(() => {
+.then(_ => {
   console.log('Notifications have been started.');
 })
 .catch(error => { console.log(error); });
 
-function handleNotifications(event) {
+function handleCharacteristicValueChanged(event) {
   var value = event.target.value;
-  var textDecoder = new TextDecoder();
+  var textDecoder = new TextDecoder(); // Used to convert bytes to UTF-8 string.
   console.log('Received ' + textDecoder.decode(value));
 }
 {% endhighlight %}
 
-The dedicated [sample](https://googlechrome.github.io/samples/web-bluetooth/notifications.html)
+The [Notifications Sample](https://googlechrome.github.io/samples/web-bluetooth/notifications.html)
 will show you to how to stop notifications with `stopNotifications()` and
 properly remove the added `characteristicvaluechanged` event listener.
 
+### Get disconnected from a Bluetooth Device
+
+In order to provide a better user experience, you may want to show a warning
+message if the `BluetoothDevice` gets disconnected to invite user to reconnect.
+
+{% highlight javascript %}
+navigator.bluetooth.requestDevice({ filters: [{ name: 'Francois robot' }] })
+.then(device => {
+  // Set up event listener for when device gets disconnected.
+  device.addEventListener('gattserverdisconnected', onDisconnected);
+
+  // Attempts to connect to remote GATT Server.
+  return device.gatt.connect();
+})
+.then(server => {...})
```

Comment:
optional nit: Moving this to the next line and putting the "..." in a comment would mean that anyone could copy and paste this code and run it without any modifications. WDYT?